spec/controllers/api/v1/va_notify_controller_spec.rb

```diff
@@ -0,0 +1,140 @@
+# frozen_string_literal: true
+
+describe Api::V1::VaNotifyController, type: :controller do
+  before do
+    Seeds::NotificationEvents.new.seed!
+  end
+  let(:api_key) { ApiKey.create!(consumer_name: "API Consumer").key_string }
+  let!(:appeal) { create(:appeal) }
+  let!(:notification_email) { create(:notification, appeals_id: appeal.uuid, appeals_type: "Appeal", event_date: "2023-02-27 13:11:51.91467", event_type: "Quarterly Notification", notification_type: "Email", notified_at: "2023-02-28 14:11:51.91467", email_notification_external_id: "3fa85f64-5717-4562-b3fc-2c963f66afa6", email_notification_status: "No Claimant Found") }
```

Comment:
```suggestion
    let!(:notification_email) do
    create(
      :notification,
      appeals_id: appeal.uuid,
      appeals_type: "Appeal",
      event_date: "2023-02-27 13:11:51.91467",
      event_type: "Quarterly Notification",
      notification_type: "Email",
      notified_at: "2023-02-28 14:11:51.91467",
      email_notification_external_id: "3fa85f64-5717-4562-b3fc-2c963f66afa6",
      email_notification_status: "No Claimant Found"
    )
  end
```
Using `do .. end` to create the block for the `let` will provide some more room to work with, and keep CodeClimate at bay!